Mapper/BaseGroupedMapper.php

```diff
@@ -257,6 +263,11 @@ public function hasOpenTab()
     abstract protected function setTabs(array $tabs);
 
     /**
+     * @return string
+     */
+    abstract protected function getContext();
```

Comment:
Could you find another name? "context" feels like "environment" (a list of variables). Maybe `getMapperName` or simply `getName` ?